README.md

```diff
@@ -18,3 +18,9 @@ Type: `less Makefile` to see other commands
 
 ### Default User password
 'pass123'
+
+### Frontend instructions
```

Comment:
As the frontend project is hidden inside a `docker` container, we cannot rely on `yarn` (or even `node`) being available in host context. These all commands should be added to `Makefile` (like `black` and `black-check`).

Also using the `Makefile`-based `help` command and describing the commands behind `##` will allow us to leave in `README` generic information, how to obtain information regarding all commands exposed by `Makefile` (to obtain it you can just run `make help`).

Reply:
I understand the argument that putting it in the makrefile would actually ease things. Yet I'm not sure how to deploy prettier without yarn or node, so if you have any ideas how to do that I could use help with that to save some time and focus on developing frontend components.